Tools/Migration/folder-options-dpct/LICENSE.txt

```diff
@@ -0,0 +1,22 @@
+Copyright Intel Corporation
```

Comment:
Is there a reason that you are not using the standard oneAPI License file? If not, you should be using this one: https://github.com/oneapi-src/oneAPI-samples/blob/master/License.txt

Reply:
It's the MIT license file. It's just been word-wrapped so it's easier to read and we've added the SPDX line and the link referencing the official MIT license file.